spec/card_display_spec.rb

```diff
@@ -0,0 +1,53 @@
+module MtgCli
+  describe CardDisplay do
+
+    let(:card) { Card.new(* {name: 'Putrefy',
+                            type: 'Instant',
+                            mana_cost: '{1}{B}{G}',
+                            text: 'Destroy target artifact or creature. It can\'t be regenerated.',
+                            artist: 'Richard Sardinha',
+                            flavor: "\"All matter, animate or not, rots when " +
+                                    "exposed to time. We merely speed up the process.\"\n" +
+                                    "—Ezoc, Golgari rot farmer",
+                            rarity: 'Rare',
+                            set_name: ['Dragon\'s Maze', 'Commander 2015'] }
+                         )}
+
+    let(:display) { CardDisplay.new(card) }
+
+    describe '#to_s' do
+      it 'displays the card traits' do
+        text = "\n\e[4mPutrefy                              1BG\e[0m\n\n" +
```

Comment:
I would use here doc instead of string concatenation. I have no idea what this string is supposed to look like.